cpp/include/cugraph_c/graph.h

```diff
@@ -94,20 +94,16 @@ void cugraph_sg_graph_free(cugraph_graph_t* graph);
  *                          be populated if error code is not CUGRAPH_SUCCESS
  * @return error code
  */
-cugraph_error_code_t cugraph_mg_graph_create(
-  const cugraph_resource_handle_t* handle,
-  const cugraph_graph_properties_t* properties,
-  const cugraph_type_erased_device_array_view_t* src,
-  const cugraph_type_erased_device_array_view_t* dst,
-  const cugraph_type_erased_device_array_view_t* weights,
-  const cugraph_type_erased_host_array_view_t* vertex_partition_offsets,
-  const cugraph_type_erased_host_array_view_t* segment_offsets,
-  bool_t store_transposed,
-  size_t num_vertices,
-  size_t num_edges,
-  bool_t check,
-  cugraph_graph_t** graph,
-  cugraph_error_t** error);
+cugraph_error_code_t cugraph_mg_graph_create(const cugraph_resource_handle_t* handle,
+                                             const cugraph_graph_properties_t* properties,
+                                             const cugraph_type_erased_device_array_view_t* src,
+                                             const cugraph_type_erased_device_array_view_t* dst,
+                                             const cugraph_type_erased_device_array_view_t* weights,
+                                             bool_t store_transposed,
+                                             size_t num_edges,
+                                             bool_t check,
+                                             cugraph_graph_t** graph,
+                                             cugraph_error_t** error);
```

Comment:
Just a comment (& question), so we currently don't have a mechanism to specify vertex list? This is necessary to handle isolated vertices.

Reply:
Good point.  We don't have that for either SG or MG graph creation.

I can add that as part of this PR, or we can create an issue to add it later.  It is properly supported by the C++ API, so it should be either to go either way.

Reply:
Added a FIXME, since we need to get this PR merged.